pkg/skaffold/docker/client.go

```diff
@@ -123,8 +131,44 @@ func newMinikubeAPIClient() (APIClient, error) {
 	return client.NewClient(host, version, httpclient, nil)
 }
 
+func detectWsl() (bool, error) {
+	if !procVerDetect {
+		procVerDetect = true
+		if _, err := os.Stat("/proc/version"); err == nil {
+			b, err := ioutil.ReadFile("/proc/version")
+			if err != nil {
+				return false, errors.Wrap(err, "read /proc/version")
+			}
+
+			if bytes.Contains(b, []byte("Microsoft")) {
+				isWsl = true
+			}
+		}
+	}
+	return isWsl, nil
+}
+
+func getMiniKubeFilename() (string, error) {
+	const winMiniKubeEnv = "SKAFFOLD_WINDOWS_MINIKUBE"
+	if found, _ := detectWsl(); found {
+		filename, envExists := os.LookupEnv(winMiniKubeEnv)
```

Comment:
Would it make more sense to use exec.LookPath with minikube.exe instead of this environment variable to find the path to Minikube?

Reply:
good idea. done.